docs/README.md

```diff
@@ -1,3 +1,32 @@
+> Write great documentation! – * [_Mobify Developer Values_](https://github.com/mobify/developer-values#️-write-great-documentation)
+
+At Mobify, documentation is written in markdown.
+
+Documentation should live as close as possible to the code it documents.
+Typically, this means including it in a `docs` folder at the root of the repo.
```

Comment:
The script in the docs below assumes a `/docs` folder. Is there any repo that this isn't true for right now?  Maybe we should just say that docs _should_ be in a `/docs` folder. #convention-over-configuration :)